apps/mobile/app.config.js

```diff
@@ -44,26 +38,57 @@ export default () => {
   return {
     expo: {
       name: 'Leather',
+      owner: 'leather-wallet',
       slug: 'leather-wallet-mobile',
       version: '2.2.0',
+      runtimeVersion: {
+        policy: 'fingerprint',
+      },
+      newArchEnabled: true,
```

Comment:
I want to say in Expo 53 this is enabled by default? Fine to keep it explicit though.